src/orquestra/sdk/_base/_driver/_ce_runtime.py

```diff
@@ -52,6 +52,37 @@ def _get_max_resources(workflow_def: WorkflowDef) -> _models.Resources:
     return _models.Resources(cpu=max_cpu, memory=max_memory, gpu=max_gpu, nodes=None)
 
 
+def _verify_workflow_resources(
+    wf_resources: _models.Resources, max_resources: _models.Resources
+):
+    # Default workflow CPU and memory defined on platform side, if Nones are passed
+    # https://github.com/zapatacomputing/workflow-driver/blob/1d4d0552e44cfd11238b823cbf66a33d4d2e8593/pkg/config/config.go#L22  # noqa
+    default_workflow_cpu = "2"
+    default_workflow_memory = "2Gi"
+    insufficient_resources = set()
```

Comment:
**suggestion**: looks like the only usage of `insufficient_resources` is to construct the exception message. `insufficient_resources` is a `set`, so the `str()` will differ between Python processes. `list` would make the messages deterministic without affecting the logic:
```suggestion
    insufficient_resources = list()
```